src/microbe_stage/EnergyBalanceInfoFull.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections.Generic;
+
+/// <summary>
+///   Info regarding a microbe's energy balance in a patch
```

Comment:
The summary comments should be made distinct between this and the simple variant.